low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -0,0 +1,360 @@
+            returncode = os.system(cmd)
+            if returncode != 0:
+                raise DeployError(returncode, "Failed to install rabbitmq-server.")
+
+            print("INFO: Done installing SSPL dependencies.")
+
+    def install_sspl_rpms(self):
+        """Install SSPL RPMS."""
+        # If local RPMS location is specified, SSPL RPMS will be
+        # installed from the speicifed path. Otherwise yum repos.
+        if self.RPMS_PATH:
+            print("INFO: Installing SSPL RPMS from local path - %s" % (self.RPMS_PATH))
+            packages = glob.glob(self.RPMS_PATH+'/cortx-*-2.*')
+            sspl_rpm_cmd = f"sudo yum install -y {' '.join(packages)}"
+            returncode = os.system(sspl_rpm_cmd)
+        else:
+            print("INFO: Installing SSPL RPMS using yum repos...")
+            sspl_cmd = "yum install -y cortx-sspl.noarch"
+            sspl_test_cmd = "yum install -y cortx-sspl-test"
+            returncode = os.system(sspl_cmd)
+            if returncode != 0:
+                raise DeployError(returncode, "Failed to install 'cortx-sspl'")
+            returncode = os.system(sspl_test_cmd)
+            if returncode != 0:
+                raise DeployError(returncode, "Failed to install 'cortx-sspl-test'")
+
+        print("INFO: INSTALLING required python3 site-packages..")
+        cmd = "python3 -m pip install -r /opt/seagate/cortx/sspl/low-level/requirements.txt"
+        _ = os.system(cmd)
+
+        print("INFO: Done installing SSPL RPMS.")
+
+    def update_config(self):
+        """Update sspl and sample global config values."""
+        from cortx.utils.conf_store import Conf
+        SSPL_BASE_DIR = "/opt/seagate/cortx/sspl"
+
+        self.sspl_config = "%s/conf/sspl.conf.%s.yaml" % (SSPL_BASE_DIR, self.PRODUCT_VERSION)
+        self.sample_global_config = "%s/conf/sample_global_cortx_config.yaml" % (SSPL_BASE_DIR)
+        if not os.path.exists(self.sspl_config):
+            raise DeployError(1, "sspl config file not found.")
+        if not os.path.exists(self.sample_global_config):
+            raise DeployError(1, "Sample global config file not found.")
+
+        print("INFO: Updating config values in config files..\n" + \
+            "SSPL_CONFIG: %s \nGLOBAL_CONFIG: %s" % (self.sspl_config, self.sample_global_config))
+        Conf.load("sspl", "yaml://%s" % (self.sspl_config))
+        Conf.load("global_config", "yaml://%s" % (self.sample_global_config))
+
+        Conf.set("sspl", "RABBITMQINGRESSPROCESSOR>username", self.RMQ_USER)
+        Conf.set("sspl", "RABBITMQEGRESSPROCESSOR>username", self.RMQ_USER)
+        if self.RMQ_PASSWD:
+            Conf.set("sspl", "RABBITMQINGRESSPROCESSOR>password", self.RMQ_PASSWD)
+            Conf.set("sspl", "RABBITMQEGRESSPROCESSOR>password", self.RMQ_PASSWD)
+            Conf.set("sspl", "LOGGINGPROCESSOR>password", self.RMQ_PASSWD)
+        Conf.set("global_config", "storage>enclosure_1>controller>primary>ip", self.CNTRLR_A)
```

Comment:
Here you can fetch enclosure_id(enclosure_1), from  cluster>{srvnode}>storage>enclosure_id

Reply:
As discussed and as commented in previous conversation.

Reply:
Agreed to use this way.
cc: @satish-seagate

Reply:
Done